common-lib/src/cmr/common/config.clj

```diff
@@ -249,5 +256,6 @@
          true)
        false))))
 
+;; TODO: Generic work after initial testing in SIT, this can be removed
```

Comment:
Please make this a sub-task or checklist item on the appropriate ticket and remove this TODO

Reply:
https://bugs.earthdata.nasa.gov/browse/CMR-8599 - although we want to leave the TODO messages so it will be easy to find the changes to be made. It will also allow us to find the needed changes if we miss a ticket.  We will remove these comments when the fix has been made.